.github/workflows/ci.yml

```diff
@@ -65,9 +63,7 @@ jobs:
         env:
           NUGET_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          dotnet-version: |
-            8.0.x
-            9.0.x
```

Comment:
is there a downside to switching to the global.json, and targeting only the dotnet-version defined there? 

I'm wondering if this change may make new versions of the dotnet-sdk not work for some people on 8.0.x

Reply:
I did a quick `dotnet publish` and tried to inspect the .nupkg.
The file structure I got was:
<img width="432" alt="Screenshot 2025-04-07 at 19 47 57" src="https://github.com/user-attachments/assets/5fde4fe1-064d-454b-8d99-849fa77e0d64" />
I am happy to assume that people on net8.0 will have access to the correct DLL since it is part of the nupkg file.

Reply:
Thanks for checking!